Examples/Navigation/NavigationApp.swift

```diff
@@ -1,16 +1,19 @@
+import Foundation
 import SwiftCrossUI
 
-enum SubjectArea: Hashable {
+enum SubjectArea: Codable {
     case science
     case humanities
 }
 
-enum ScienceSubject: Hashable {
-    case physics
-    case chemistry
+struct Science {
```

Comment:
I assume this change is just for testing decoding of non-top-level types?

Reply:
correct